src/LanguageServer.php

```diff
@@ -190,6 +191,7 @@ public function initialize(ClientCapabilities $capabilities, string $rootPath =
             $this->projectIndex = new ProjectIndex($sourceIndex, $dependenciesIndex, $this->composerJson);
             $stubsIndex = StubsIndex::read();
             $this->globalIndex = new GlobalIndex($stubsIndex, $this->projectIndex);
+            $initializationOptions = $initializationOptions ?? new Options;
```

Comment:
It's up to us to defined -  I am just saying it's a bit confusing that they are different. It would be much easier for a client if we just document please pass the `didChangeConfiguration` settings as `initializationOptions`